BinaryEncoding.md

```diff
@@ -372,9 +478,10 @@ out of range, `br_table` branches to the default target.
 | `f32.const` | `0x13` | value : `uint32` | a constant value interpreted as `f32` |
 | `get_local` | `0x14` | local_index : `varuint32` | read a local variable or parameter |
 | `set_local` | `0x15` | local_index : `varuint32` | write a local variable or parameter |
-| `call` | `0x16` | argument_count : `varuint1`, function_index : `varuint32` | call a function by its index |
+| `get_global` | `0x18` | global_index : `varuint32` | read a global variable |
+| `set_global` | `0x19` | global_index : `varuint32` | write a global variable |
+| `call` | `0x16` | argument_count : `varuint1`, function_index : `varuint32` | call a function by its [index](Modules.md#function-index-space) |
```

Comment:
Since we're going to defragment this opcode space anyway, I'll change these codes.